### ISSUE TITLE
"In Vue 2.7, when using Tailwind CSS 2.0 version and the Tailwind Element component without enabling the JIT mode, the ripple effect of buttons may not be successful."

### ISSUE THREAD

**Issue by Eagon7** [mdbootstrap/TW-Elements#1463]
**Describe the bug**
No, using Tailwind Element in Vue 2.7 does not require you to enable JIT mode. However, if you want to use certain features, such as the button ripple effect, then you may need to enable JIT mode.
ata-te-ripple-init cannot be compiled

**Comment by Eagon7** [comment]
In Tailwindcss JIT mode, it is very slow to rebuild the project every time it is saved.

**Comment by juujisai** [comment]
Hi. This is an older issue so I don't know whether that was fixed in `v1.0.0-beta2` but I do not see the issue right now. I created a new app with vue 2.7 and latest tailwind (our components are using some of the newer things offered by tailwind, so the most recent version the better). 

I've used the first examples from Ripple and Datepicker in my test. I found out while dealing with issues experienced in nuxt and next, that sometimes importing components with use of dynamic imports inside `mounted` hooks helps. 
For example like that: 
```
<script>
export default {
  name: "App",
  components: {},
  mounted: async function () {
    const { Datepicker, Input, Ripple, initTE } = await import("tw-elements");

    initTE({ Datepicker, Input, Ripple });
  },
};
</script>
```

Let us know if the issue is still here and whether my solution helped.
